### PR TITLE
(RE-12428) Add support for redhatfips-7

### PR DESCRIPTION
**Pull request by sarameisburger**
Needs testing

### CODE REVIEW

**Comment by sarameisburger** [comment]
Successful build of puppetserver https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_various-packaging-jobs_packaging-os-clj_lein-ezbake-generic/1803/

File: resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ packager: 'puppet'
 gpg_key: '7F438280EF8D349F'
 sign_tar: FALSE
 # a space separated list of mock configs
-final_mocks: 'pupent-el6-x86_64 pupent-el7-x86_64 pupent-el8-x86_64 pupent-sles12-x86_64'
+final_mocks: 'pupent-el6-x86_64 pupent-el7-x86_64 pupent-el8-x86_64 pupent-redhatfips-7-x86_64 pupent-sles12-x86_64'
```
Comment (mwaggett) [inline resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml]: I know packaging (sort of) uses these values, but I'm not totally sure how/where. The `pupent` thing is something I've only ever seen in build_defaults, so I'm not totally sure if this is formatted correctly or not (e.g. should this be `pupent-redhatfips7-x86_64` (no space between platform and version) to match the rest??), or whether there needs to be some additional setup for this.

Comment (sarameisburger) [reply resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml]: Yeah I don't really know why they're formatted differently from the foss build_defaults but will update!

Comment (shrug) [reply resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml]: Legacy from old build infra that we no longer use, but packaging still parses and uses the names. It's from the old mock builds: https://github.com/rpm-software-management/mock/wiki

Comment (sarameisburger) [reply resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml]: Ah gotcha. Updated.

**Pull request merged**